rero_ils/utils.py

```diff
@@ -65,3 +65,25 @@ def send_mail(subject, recipients, template, language, **context):
 def unique_list(data):
     """Unicity of list."""
     return list(dict.fromkeys(data))
+
+
+def get_agg_config(index_name, field):
+    """Get Elasticsearch aggragation term configuration.
```

Comment:
s/aggragation/aggregation/

All occurences in this file.

Reply:
done